src/richie/apps/search/signals.py

```diff
@@ -0,0 +1,70 @@
+"""Update Elasticsearch indexes each time a page is modified."""
+from django.core.exceptions import ObjectDoesNotExist
+from django.db import transaction
+
+from richie.apps.courses.models import Category, Course, Organization
+from richie.apps.search.indexers import ES_INDICES
+from richie.apps.search.index_manager import richie_bulk
+
+
+def update_course(course, _language):
```

Comment:
Nice design for all these `update_` methods 👍

Reply:
pylint did like it :disappointed: Let me try another design...

Reply:
Done.

Reply:
I think the previous one was more elegant but this is alright too